nurseconnect/views.py

```diff
@@ -1,12 +1,113 @@
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.models import User
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
 from django.utils.translation import get_language_from_request
+from django.views.generic import FormView
+from django.views.generic import TemplateView
 
 from molo.core.utils import get_locale_code
 from molo.core.models import ArticlePage
+from molo.profiles.views import RegistrationView, MyProfileEdit
+from nurseconnect import settings
+from nurseconnect.forms import NurseConnectRegistrationForm, NurseConnectForgotPasswordForm, \
+    NurseConnectResetPasswordForm, NurseConnectEditProfileForm
 from wagtail.wagtailsearch.models import Query
 
 
+class HomeView(TemplateView):
+    template_name = 'core/main.html'
+
+    def render_to_response(self, context, **response_kwargs):
+        # username = self.request.GET.get('user')
+        # token = self.request.GET.get('token')
+
+        # if not username or not token:
+        #     return HttpResponseForbidden()
+        #     template_name = 'core/home_page_before_login.html'
+        # context['form'].initial.update({
+        #     'username': username,
+        #     'token': token
+        # })
+
+        return render(self.request, self.template_name, context)
+
+
+class NurseConnectRegistrationView(RegistrationView):
+    form_class = NurseConnectRegistrationForm
+
+    def form_valid(self, form):
```

Comment:
Try iterating the model fields and using the field names as the key for cleaned data and save the object after iteration.